lib/storage-service.js

```diff
@@ -65,19 +65,31 @@ class StorageService {
    * @returns {Promise}
    */
   lookup(token, secret, ProviderType) {
-    return this._getLeaseManager(token, secret, ProviderType).then((manager) => {
+    let manager = null;
+
+    return this._getLeaseManager(token, secret, ProviderType).then((m) => {
+      manager = m;
+
       if (manager.status === STATUS.READY) {
+        StorageService.logManagerEvent('INFO', 'Manager was able to lookup a secret.', manager);
```

Comment:
Do we want to log `manager` here? That will include `manager.data` which is the secret. Maybe use `StorageService.logManagerEvent` instead?